calico-enterprise_versioned_docs/version-3.16/reference/host-endpoints/connectivity.mdx

```diff
@@ -34,7 +34,7 @@ appropriate values for your deployment.
 
 <!-- -->
 
-```bash
+```bash 
 cat <<EOF | calicoctl create -f -
 - apiVersion: projectcalico.org/v3
```

Comment:
Is this one passing the test? I get the feeling that the hyphen that starts the first line of the YAML isn't right.